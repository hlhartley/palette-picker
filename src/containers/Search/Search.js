import React, { Component } from 'react';

export class Search extends Component {
  render() {
    return (
      <div className="Search">
        <form className="search-form">
          <label htmlFor="search" className="search-label">Search by hexcode, palette, or project:</label>
          <input type="text" name="search" className="search-input"></input>
          <button type="submit">Search</button>
        </form>
      </div>
    );
  }
}

export default Search;
