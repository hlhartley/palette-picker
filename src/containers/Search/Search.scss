.Search {
    border: 2px solid powderblue;
}