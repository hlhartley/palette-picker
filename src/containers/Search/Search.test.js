import React from 'react';
import { shallow } from 'enzyme';
import { Search } from './Search';

describe('Search', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Search/>
      )
    });

  describe('Search component', () => {
    it('should properly render the component elements', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});