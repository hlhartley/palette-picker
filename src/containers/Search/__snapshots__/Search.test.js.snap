// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Search component should properly render the component elements 1`] = `
<div
  className="Search"
>
  <form
    className="search-form"
  >
    <label
      className="search-label"
      htmlFor="search"
    >
      Search projects by hexcode:
    </label>
    <br />
    <input
      className="search-input"
      name="search"
      type="text"
    />
    <button
      className="search-button"
      type="submit"
    >
      Search
    </button>
  </form>
</div>
`;
